src/globus_sdk/_testing/data/compute/v2/delete_function.py

```diff
@@ -1,6 +1,6 @@
 from globus_sdk._testing.models import RegisteredResponse, ResponseSet
 
-from ._common import FUNCTION_ID
+from .._common import FUNCTION_ID
```

Comment:
If these are moving, we'll need to update [the resolution of bound client methods](https://github.com/globus/globus-sdk-python/blob/5d96deb900e3b00e779c3cf0e0d4b0c91c180ab0/src/globus_sdk/_testing/registry.py#L36) to handle it.

I don't see a way to do it generically, so I would say we need a specialized codepath specifically for these multiversioned clients. (The alternative is to design and add a new generic path for this, but I'm not sure it's worth it.)

Reply:
I also think that we need to see how this will work with autodoc for this new bimodal suite of fixtures:
https://github.com/globus/globus-sdk-python/tree/main/docs/testing/methods

I haven't gotten to it yet, but we really need a `compute.rst` in that directory. I'm pretty sure it will work fine to list both the v2 and v3 clients, but we should add them and confirm that docs build correctly.

Reply:
As discussed out-of-band, the tests passed on my machine due to a stale `build/` directory.

I'll work on a fix for the client method resolution.

Reply:
I'm going to mark this particular thread as resolved.
(1) We tinkered with that "versioned test fixture resolution" behavior, so that's technically solved.
(2) The nasty dev-time bug which made this hard to spot in local testing is solved via the new `toxfile.py`, so the follow-up is done.